srtcore/common.h

```diff
@@ -462,6 +462,18 @@ class CTimer
 
    static uint64_t getCPUFrequency();
 
+   /// Shortcut for converting ticks to microseconds
+   static uint64_t tk2us(uint64_t tk)
+   {
+       return tk/getCPUFrequency();
```

Comment:
Add spaces: `tk / getCPUFrequency();`

Reply:
Could be. I've been using such convention that spaces are around + and - operators, but not around * and /, which should improve visibility of them in the matter of priorities.